xbmc/interfaces/legacy/Control.cpp

```diff
@@ -150,7 +150,7 @@ namespace XBMCAddon
     {
       if (!pGUIControl) return NULL;
 
-      LOCKGUI;
+      XBMCAddonUtils::GuiLock(languageHook);
```

Comment:
That does seem to do the trick indeed but I don't understand why as it's not needed in the master branch (original PR). Before the removal of the offScreen argument it was building fine.
What do to with it? Update the PR with your proposed change and leave it like that ? I'm not exactly a C++ hero (but learning slowly) so I want to understand what's happening.